ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchaserInfoHybridAdditionsTests.swift

```diff
@@ -22,15 +23,15 @@ class PurchaserInfoHybridAdditionsTests: QuickSpec {
                     let url = URL(string: urlPath)
                     purchaserInfo.stubbedManagementURL = url
                     
-                    let dictionary = purchaserInfo.dictionary()
-                    expect(dictionary?["managementURL"] as? String) == urlPath
+                    let dictionary = purchaserInfo.dictionary
+                    expect(dictionary["managementURL"] as? String) == urlPath
```

Comment:
looks like we don't need the .absoluteSting then

Reply:
see my other comment, it turns out that Quick wasn't running correctly locally 🤦